### PR TITLE
Hero video and images

### LINKED ISSUE #6675: Storytelling Wagtail Component: Header Image/ Hero Video Block
### Intro
To allow for more compelling storytelling on the Wagtail blog pages, we want to implement a larger hero image/video block section within the Wagtail blog template.

---

###  Design -> Dev Handoff Checklist

- [x] Design has been finalized 
- [x] Link to original design ticket (original design ticket is in #828 (https://app.zenhub.com/workspaces/mofo-engagement-585335eab729771d0736378d/issues/mozillafoundation/design/828)
- [x] [Link to Redpen](https://redpen.io/p/wu5fe36cd1366a0cad)
- [x] [Link to Figma file](https://www.figma.com/file/OTFsYEpkuXcR6HgowgDlyO/Wagtail-Storytelling-Components?node-id=0%3A1)
- [x] Desktop, tablet, and mobile mockups are included
- [ ]Assets are included. Link (https://drive.google.com/open?id=15XTjgbU_UpnGBEGvslNe5AvX4kMGONM3&authuser=natalie%40mozillafoundation.org&usp=drive_fs)
- [x] Hover state for elements are designed ([the prototype is here for the play/pause states on the page](https://www.figma.com/proto/OTFsYEpkuXcR6HgowgDlyO/Wagtail-Storytelling-Components?node-id=55%3A766&scaling=min-zoom&page-id=55%3A700) and the designed [Figma file is here](https://www.figma.com/file/OTFsYEpkuXcR6HgowgDlyO/Wagtail-Storytelling-Components?node-id=55%3A700) for reference

Please reach out to @natalieworth if you have any questions or need clarification! :)

@benhohner I'm going to leave the estimate up to your team. I'm not sure what's best or how long each issue will take! :)

### PR DESCRIPTION
**Pull request by danielfmiranda**
Closes #6675 

**Link to sample test pages**:
Test Page with Video Header: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/
Test Page with Image Header: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/call-budget-yard-impact-art-trial/
Test Page with Neither: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/test-glass-oil-book-color-opportunity/

**Steps to Test**:

1. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/, video should load at the top and start playing immediately.
2. Video should be responsive, and play/pause button should work as expected.
3. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/call-budget-yard-impact-art-trial/, there should be a responsive image at the top of the page. 
4. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/test-glass-oil-book-color-opportunity/, it should look like previous blog posts.

**NOTE:** Once PR is approved, we will need to add vimeo to the allowed media sources in heroku.
## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible]

### CODE REVIEW

**Review by sabrinang — CHANGES_REQUESTED** [review]
Responsive images and video work on the pages as well as the stop/play video feature.

I think the height shouldn't be taller than the home's hero and if they match these hero banner assets could be reusable between them and it gives one less different dimension to consider.
Screen Shot 2021-07-15 at 11 23 06 AM

In the CMS, should there be some validation feedback here if both are entered or it could be formatted in a way that it is clear you can do either video or image but not both. Currently, if the editor fills in both fields the video shows. 
image

**Comment by danielfmiranda** [comment]
Hi @sabrinang! I have updated the size of the hero video/image to match the size of the banner on the homepage, as well as updated the layout of the CMS fields and added some validation. They are now both under a "hero video/image" section, and it will not let users save the page with a video and image set, just one or the other. 

These changes are now live and ready for re-review, thanks!

**Review by sabrinang — APPROVED** [review]
Thanks for making the updates to the height and layout in the CMS Daniel, looks good.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_page_heroguts.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% load wagtailcore_tags wagtailimages_tags i18n static%}
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_page_heroguts.html:1]:
```suggestion
{% load wagtailimages_tags %}
```

**Review by KalobTaulien — APPROVED** [review]
Approving. One small commit suggestion (but please 2x check it first)

**Comment by danielfmiranda** [comment]
@KalobTaulien thanks for the suggestion! I have tested it out on my local machine and can confirm that it brings no undesired side effects, so I have made those changes and commited them to the repo.

 I also found that this repo was experiencing the pause button shifting issue that you mentioned in #7103, so I have added the fix to this PR as well.

**Pull request merged**